frontend/javascripts/oxalis/view/right-menu/trees_tab_view.js

```diff
@@ -322,7 +322,19 @@ class TreesTabView extends React.PureComponent<Props, State> {
   };
 
   showDeleteGroupModal = (id: number) => {
-    this.setState({ groupToDelete: id });
+    if (!this.props.skeletonTracing) return;
+
+    const { trees, treeGroups } = this.props.skeletonTracing;
+    const treeGroupToDelete = treeGroups.find(el => el.groupId === id);
+    const groupToTreesMap = createGroupToTreesMap(trees);
```

Comment:
Did you test to delete a group which only contains another group (which contains a tree)? Like this:
```
Group A
  Group B
    Tree 1
```

In that case, a modal should appear. If it doesn't work right now, you might want to use a helper function like `callDeep` or similar (have a look at `frontend/javascripts/oxalis/view/right-menu/tree_hierarchy_view_helpers.js`).

Reply:
As I'm checking whether the group has a child group, I'm guarding against that case, but is this too strict?

Reply:
So should it be allowed to delete two empty groups like:
```
Group A
  Group B
```

Reply:
> As I'm checking whether the group has a child group, I'm guarding against that case

Ah, sorry, my bad! I missed that.

> So should it be allowed to delete two empty groups like:
> 
> ```
> Group A
>   Group B
> ```

I think, that's not necessary :)